client/app/vps/dashboard/vps-dashboard.controller.js

```diff
@@ -167,6 +189,17 @@ class VpsDashboardCtrl {
                 callback: () => this.VpsActionService.monitoringSla(this.serviceName, !this.vps.slaMonitoring),
                 isAvailable: () => !this.loaders.init
             },
+            // orderAdditionalDiskOption: {
```

Comment:
dead code

Reply:
this comment is what it should be, but as discussed with @Mboulianne  i cannot set it until available works.

Reply:
I've made a fix in this PR https://github.com/ovh-ux/ovh-manager-cloud/pull/330/files
should be available in master during the day